backend/src/email/index.js

```diff
@@ -0,0 +1,35 @@
+const nodemailer = require('nodemailer')
+const pug = require('pug')
+
+const transport = nodemailer.createTransport({
+  host: process.env.EMAIL_HOST,
+  port: process.env.EMAIL_PORT,
+  auth: {
+    user: process.env.EMAIL_USERNAME,
+    pass: process.env.EMAIL_PASSWORD,
+  },
+})
+
+async function send(payload) {
+  try {
+    await transport.sendMail(payload)
+  } catch (err) {
+    console.log(err)
+    throw new Error('Email service is down. Please try again later')
+  }
+}
+
+async function sendEmailVerification(email, name, verificationUrl) {
+  await send({
+    from: '"QuoVote Team" <noreply@quo.vote>',
+    to: email,
+    subject: 'Verify your QuoVote account',
+    html: pug.renderFile(`${__dirname}/templates/verify-email.pug`, {
```

Comment:
Check if that's getting covert to raw text as well, if not provide it seperately